packages/ember-htmlbars/tests/integration/component_invocation_test.js

```diff
@@ -595,367 +66,14 @@ QUnit.test('moduleName is available on _renderNode when no layout is present', f
     }
   }));
 
-  view = EmberView.extend({
+  component = Component.extend({
     [OWNER]: owner,
     layout: compile('{{#sample-component}}Derp{{/sample-component}}', {
       moduleName: templateModuleName
     })
   }).create();
 
-  runAppend(view);
-});
-
-QUnit.test('{{component}} helper works with positional params', function() {
```

Comment:
Only partly, thought about it but then decided it would be best to do that later once this is accepted and I had the chance to talk to you what would be the best way to approach this (most tests in this file should be shareable between `curly-components` and `dynamic-components` tests). I thought this file was already too big to review so adding more complexity to it would not be worth it in a first review. My apologies!